EIPS/eip-3220.md

```diff
@@ -0,0 +1,121 @@
+---
+eip: 3220
+title: Crosschain Identifier Specification
+author: Weijia Zhang (@weijia31415), Peter Robinson (@drinkcoffee)
+discussions-to: https://ethereum-magicians.org/t/eip-3220-crosschain-id-specification/5446
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-10-21
+---
+
+## Simple Summary
+
+To provide better identification, description, and discovery of blockchains in cross-chain operations, this specification defines the syntax and methods for a cross-chain identification that is self-verifying within the blockchain, identifiable with forked blockchains, and embedded with essential information for the blockchain.
```

Comment:
Simple Summary should be shorter, think subject line of an email or title line of a GitHub pull request or forum thread.
```suggestion
A self-verifying unique blockchain identifier that deals with forks.
```